include/openenclave/internal/utils.h

```diff
@@ -34,6 +34,18 @@ OE_INLINE uint64_t oe_round_u64_to_pow2(uint64_t n)
     return x + 1;
 }
 
+OE_INLINE bool oe_is_pow2(size_t n)
+{
+    return (n != 0) && ((n & (n - 1)) == 0);
```

Comment:
Fancy

Reply:
It was in existing code and got put into a util routine. Yes, this is the elegant way to do is power.